lib/tessel/deploy.js

```diff
@@ -780,13 +760,32 @@ actions.tarBundle = function(opts) {
     });
   });
 
+  logs.info('Building project.');
+
+
   // Both the --slim and --full paths will use a copy of the
   // project to bundle. This allows us to be destructive
   // with the files, but without directly tampering with the
   // project files themselves.
   var tempBundleDir = fsTemp.mkdirSync();
+  var common = ['node_modules', 'package.json', opts.resolvedEntryPoint];
+  var assets = fs.readdirSync(target).filter(entry => (common.indexOf(entry) === -1));
 
-  logs.info('Building project.');
+  if (assets.length) {
+    logs.warn(
+      tags.stripIndent `
+        The following items were found in your project directory, but not deployed.
+
+          ${assets.join('\n')}
+
+        If you need to deploy them to your Tessel 2, you must create a .tesselinclude
+        file that lists the items you wish to deploy with your application.
+
+        You may also create a .tesselignore file to list items that should be ignored
+        by the deployment process.
```

Comment:
add a line that tells you where to go for more info?

Also, not sure whether the .tesselignore note is necessary here